bevy_lint/action.yml

```diff
@@ -0,0 +1,31 @@
+name: Install `bevy_lint`
+author: BD103
+description: |
+  Installs the Bevy linter and its required toolchain.
+
+  This action currently uses Rustup to install the required toolchain components, and it uses
+  `cargo install` to install the linter itself. The linter will be installed to `~/.cargo/bin` and
+  will be available through the `PATH` by default.
+
+  Note that this currently will override the default Rustup toolchain to nightly. You can fix this
+  by calling `rustup default` after this action is run.
+runs:
+  using: composite
+  steps:
+    - name: Install Rust toolchain and components
+      uses: dtolnay/rust-toolchain@master
```

Comment:
What happens if the user's workflow already includes this step beforehand? Nothing or does it break?

Reply:
It installs this toolchain alongside the one they previously installed. It also runs `rustup default`, which is [unfortunately not something that can be disabled](https://github.com/dtolnay/rust-toolchain/pull/144), so the new `nightly-2025-XX-XX` toolchain will be made the default.